### DISCUSSION TITLE
Feature: Automatically uncheck "feature this clip"

### DISCUSSION THREAD

**Discussion by ScrubN** [night/betterttv#6928] [Ideas]
When clipping channels in which you are an editor or the streamer, there is a checkbox to automatically feature new clips immediately after creation.

It would be awesome if betterttv had a setting to automatically uncheck this so I don't need to every time I clip something.

**Comment by night** [comment]
Closing as a duplicate of #6307. Please add your upvote and any extra detail over there so feedback stays consolidated.
